client/src/styles/fluent.scss

```diff
@@ -156,24 +157,24 @@ $fluent-color-localised: rgb(102, 15, 242);
   }
 }
 
-.action-group-COPY_FROM .action-menu__toggle:before,
-.action-group-COPY_TO .action-menu__toggle:before {
-  font-family: inherit !important;
+.action-group-COPY_FROM .action-menu__toggle:before, // sass-lint:disable-line class-name-format
+.action-group-COPY_TO .action-menu__toggle:before { // sass-lint:disable-line class-name-format
+  font-family: inherit !important;// sass-lint:disable-line no-important
   line-height: 1em;
-  font-size: 1.0em;
+  font-size: 1em;
 }
 
 // Gridfield menu items
-.action-group-COPY_FROM .action-menu__toggle:before {
+.action-group-COPY_FROM .action-menu__toggle:before {// sass-lint:disable-line class-name-format
   // force ... to turn into actual title
   // @todo i18n this
-  content: "Copy from" !important;
+  content: "Copy from" !important;// sass-lint:disable-line no-important
```

Comment:
This was a really ugly work-around because I couldn't update the button labels in react menu.

Reply:
Probably worth adding that as a comment above it - it's always nice when looking at scss which has exceptions to see comments right there telling you _why_ it's an exception.

Reply:
Yeah it was '...' which is the default label for 'more actions'.

Reply:
Ahh that makes sense haha ignore me then, I just didn't understand that comment that was already there 😅